config/database.yml.travis

```diff
@@ -3,7 +3,7 @@ default: &default
   host: localhost
   pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
   timeout: 5000
-  username: postgres
```

Comment:
Jangan diubah di `database.yml.travis`

Reply:
travisnya ttp pake username: postgres?
Aku sudah gitignore database.yml jd klo misalnya clone baru harus ditambah sendiri database.yml nya but itu just copy paste dari database.yml.travis sih~

Otherwise, klo checkout -b dari master should not be a problem